_data/tutorials.yaml

```diff
@@ -673,3 +673,22 @@ sliding-windows:
     ruby: disabled
     scala: disabled
     swift: disabled
+
+schedule-ktable-ttl:
+  title: "How to implement TTLs with KTable data"
+  meta-description: "How to implement TTLs with KTable data"
+  slug: "/schedule-ktable-ttl"
+  question: "Do you have a requirement to clean up KTable data (in the topic and in the state store) based on TTL"
```

Comment:
suggestion - change the `question` section to `How do I implement a TTL clean-up on a KTable?` or something like that.